packages/prerender/src/runPrerender.tsx

```diff
@@ -177,6 +182,17 @@ export const runPrerender = async ({
   // Prerender specific configuration
   // extends projects web/babelConfig
   registerWebSideBabelHook({
+    plugins: [
+      [
+        'babel-plugin-module-resolver',
+        {
+          // gets the config from `web/tsconfig.json`
+          // then add the the user defined aliases
+          alias: getPathsFromWebTsconfig(),
+          loglevel: 'silent', // to silence the unnecessary warnings
```

Comment:
What are the unnecessary warnings?

Reply:
It will tell me it can't resolve the components, but everything works.

```sh
rw-prerender-aliases|main⚡ ⇒ yarn rw build
✔ Generating Prisma Client...
✔ Generating Prisma Client...
✔ Verifying graphql schema...
✔ Building API...
✔ Cleaning Web...
✔ Building Web...
Starting prerendering...
⠋ Prerendering / -> web/dist/index.html
Could not resolve "src/components/ui/Emoji/Emoji" in file /Users/esteban/files/github/esteban-url/rw-prerender-aliases/web/src/pages/HomePage/HomePage.tsx.
Could not resolve "src/components/ui/Hello/Hello" in file /Users/esteban/files/github/esteban-url/rw-prerender-aliases/web/src/p
✔ Prerendering / -> web/dist/index.html
```

When I leave it on, it's quiet, and everything still works

```sh
rw-prerender-aliases|main⚡ ⇒ yarn rw build
✔ Generating Prisma Client...
✔ Generating Prisma Client...
✔ Verifying graphql schema...
✔ Building API...
✔ Cleaning Web...
✔ Building Web...
Starting prerendering...
⠋ Prerendering / -> web/dist/index.html
Could not resolve "src/components/ui/Emoji/Emoji" in file /Users/esteban/files/github/esteban-url/rw-prerender-aliases/web/src/pages/HomePage/HomePage.tsx.
Could not resolve "src/components/ui/Hello/Hello" in file /Users/esteban/files/github/esteban-url/rw-prerender-aliases/web/src/p
✔ Prerendering / -> web/dist/index.html```

Reply:
the newest commit got rid of this changes all together. resolving